src/internal/AsyncSubject.ts

```diff
@@ -14,10 +14,10 @@ export class AsyncSubject<T> extends Subject<T> {
 
   /** @internal */
   protected _checkFinalizedStatuses(subscriber: Subscriber<T>) {
-    const { hasError, _hasValue, _value, thrownError, isStopped } = this;
+    const { hasError, _hasValue, _value, thrownError, isStopped, _isComplete } = this;
     if (hasError) {
       subscriber.error(thrownError);
-    } else if (isStopped) {
+    } else if (isStopped || _isComplete) {
```

Comment:
Basically, there's a temporal space where we're `_isComplete`, but not `isStopped` while we're nexting to consumers in our `complete()` method. If someone subscribes during that, we end up hitting this code, and if we're only checking `isStopped` it's not "stopped" yet, so we didn't notify the consumer.